### ISSUE TITLE
CalledProcessError are really hard to debug (or how to catch CalledProcessError)

### ISSUE THREAD

**Issue by olivierdalang** [incuna/django-wkhtmltopdf#107]
Hi !

I generate fairly complex PDFs (with openlayer maps making ajax calls) and have trouble debugging them. I often get `CalledProcessError` because of different things, and the only way I found to see the actual command line errors that happened is to copy and paste the display command in the terminal.
But it is time consuming and some other error can happen (permissions error, etc.)

Is there a way to display the command line errors of a failing PDFTemplateResponses call directly in the browser ?

I expected something like this: 

```
try:
        return PDFTemplateResponse(...)
except Exception as e:
        return HttpResponse('didnt work ! '+str(e))
```

But the exception isn't catched !

Thanks !

Olivier

**Comment by johnraz** [comment]
Could be late on the game but did you try to put the following in you `settings.py`:

```
WKHTMLTOPDF_CMD_OPTIONS = {
    'quiet': False,
}
```

**Comment by olivierdalang** [comment]
Hi,

I tried this, and also 
```
WKHTMLTOPDF_CMD_OPTIONS = {
    'quiet': False,
}
```
But I still get the same exception message :
```
CalledProcessError at /attestations/pdf_approbation/47ec9cd4-4348-48b4-87bf-864306634f06/
Command '['/home/olivier/wkhtmltox/bin/wkhtmltopdf', '--debug-javascript', '--enable-javascript', '--encoding', 'utf8', '--footer-html', '/tmp/wkhtmltopdfmoxotjbk.html', '--header-html', '/tmp/wkhtmltopdfh0ea2nsx.html', '--javascript-delay', '2500', '--no-stop-slow-scripts', '/tmp/wkhtmltopdfd_ilam2a.html', '-']' returned non-zero exit status 1
```
It just says the exit status is 1, but no way to see the actual command from the output  in the browser.

**Comment by devrijal** [comment]
Hey, I got the same error , non-zero exit retcode -6, 

What causing this problem ? Permission I set was 777 for sure there is no permission error, but with manual command line, it work fine (with no any global option)

**Comment by johnraz** [comment]
I'm gonna assume that this is improved and fixed by #134.
If it's not the case please re-open.

**Comment by DharienNightroad** [comment]
I am experiencing this error with Ubuntu 16.04, wkhtmltopdf 0.12.2.4

`Command '['/usr/bin/wkhtmltopdf', '--encoding', u'utf8', '--ignore-404', '--margin-top', u'2', '--quiet', '/tmp/wkhtmltopdfQLvjHC.html', '-']' returned non-zero exit status 1`

My settings
`WKHTMLTOPDF_CMD = '/usr/bin/wkhtmltopdf'

WKHTMLTOPDF_CMD_OPTIONS = {
    'ignore_404': True,
    'quiet': True,
}`

The View that calls this
`class CollaboratorPDFView(LoginRequiredMixin, DetailView):
    template = 'pdf-reports/gdh/collaborator_pdf_report.html'
    context = {'title': 'Colaborador pdf'}
    model = Collaborator

    def get(self, request, *args, **kwargs):
        collaborator = self.get_object()
        self.context['collaborator'] = collaborator

        response = PDFTemplateResponse(request=request,
                                       template=self.template,
                                       filename="collaborador_" + str(collaborator.id) + ".pdf",
                                       context=self.context,
                                       show_content_in_browser=False,
                                       cmd_options={'margin-top': 2, }
                                       )
        return response`
